gtk/src/auto/accel_group.rs

```diff
@@ -84,14 +87,17 @@ pub trait AccelGroupExt: 'static {
         f: F,
     ) -> SignalHandlerId;
 
+    #[doc(alias = "accel-changed")]
     fn connect_accel_changed<F: Fn(&Self, u32, gdk::ModifierType, &glib::Closure) + 'static>(
         &self,
         detail: Option<&str>,
         f: F,
     ) -> SignalHandlerId;
 
+    #[doc(alias = "notify::is-locked")]
     fn connect_property_is_locked_notify<F: Fn(&Self) + 'static>(&self, f: F) -> SignalHandlerId;
```

Comment:
@sdroege : ... For some signal, the name is a bit different though. This is the name we pass to the C API, so I guess it's ok.

Reply:
Different in what sense? But yes, always put the C signal name there.

However: the specific case of the notify signal should IMHO just get the doc alias for the property name. It's part of the property.

Reply:
> However: the specific case of the notify signal should IMHO just get the doc alias for the property name. It's part of the property.

This is the difference I was referring too :) I'm on it.

Reply:
Good that you mention this. This also needs another rename :) `connect_property_is_locked_notify` -> `connect_is_locked_notify`. No point in having the word property in there if we don't have it for the getter/setter anymore.

Reply:
Indeed!

Reply:
I just hope, this will not lead me to perform name conflict resolution like for getters...

Reply:
That would surprise me very much.

Reply:
You were right. It went smoothly :)

We get this now:

```rust
    #[doc(alias = "is-locked")]
    fn connect_is_locked_notify<F: Fn(&Self) + 'static>(&self, f: F) -> SignalHandlerId {
```
[link to code](https://github.com/fengalin/gtk-rs/blob/9451abcdb734624e5b2b888facab6f9f746b63d8/gtk/src/auto/accel_group.rs#L264)

Reply:
That's awesome